### DISCUSSION TITLE
I occasionally see the message "Archive authentication DISABLED."

### DISCUSSION THREAD

**Discussion by enricocavalli** [borgbackup/borg#7895] [Q&A] [answered]
I recently upgraded to latest borg 1.2.6 on both client and server (I tend to stay always with latest stable version). I did the reccomended checks in upgrade notes and all my  archives in every repo I have seem ok from TAM perspective.

Occasionally, I see these messages during prune phase:

Archive authentication DISABLED.
TAM-verified archive

Is this ok ? I don't understand exactly if it's normal or not. 
Thank you for your help.

**Comment by ThomasWaldmann** [comment, marked as answer]
That's a debug level message and you can ignore it - you shouldn't see it unless you run with --debug.

borg versions older than 1.2.5 and 1.2.6 never verified archive TAMs.

To make borg not abort when touching existing repos (before the upgrade steps are completed), it needs to have archive TAM authentication disabled at a few places (while enforcing it at other places).
